packages/odyssey-react-mui/src/ScreenReaderText.tsx

```diff
@@ -20,6 +20,7 @@ export type ScreenReaderTextProps = {
    * The visually-hidden text.
    */
   children: ReactNode;
+  id?: string;
```

Comment:
This change is left over from my initial effort to move the screen reader text out of the actual elements and use `aria-describedby`. Figured we will eventually make that change so, may as well leave this in